package.json

```diff
@@ -159,12 +159,12 @@
     {
       "path": "lib/components/internal/plugins/index.js",
       "brotli": false,
-      "limit": "11 kB"
+      "limit": "14 kB"
```

Comment:
2.75 extra kilobytes after adding the metrics import. Same increase propagated to the `widget-exports.js` below